assets/css/sensei-course-theme/course-navigation.scss

```diff
@@ -0,0 +1,132 @@
+.sensei-lms-course-navigation {
+
+	--gray: #787C82;
+	--primary-color: var(--theme-color, #30968B);
+
+	// Reset theme CSS.
+	* {
+		font-family: inherit;
+		font-size: initial;
+		color: initial;
+		border: initial;
+		font-weight: initial;
+		background: initial;
+		padding: initial;
+		margin: initial;
+		transition: all 300ms;
+	}
+
+	a:hover, a:hover * {
+		color: var(--primary-color);
+		fill: var(--primary-color);
+	}
+
+	button:hover, button:focus {
+		background: initial;
+	}
+
+	@import '../../blocks/shared-style';
+
+	font-weight: 300;
+	font-size: 13px;
+
+	&__modules:not(:empty) ~ &__lessons:not(:empty) {
+		border-top: 1px dotted #ccc;
+		margin-top: 6px;
+		padding-top: 6px;
+	}
+
+}
+
+.sensei-lms-course-navigation-module {
+	&__header {
+		display: flex;
+		align-items: flex-start;
+		margin: 12px 0;
+
+		.sensei-collapsible__toggle {
+			svg {
+				width: 24px;
+				height: 24px;
+			}
+
+			&.collapsed {
+				transform: rotate(180deg);
+			}
+
+			height: 24px;
+		}
+	}
+
+	&__title {
+		flex: 1;
+
+		a {
+			color: var(--primary-color);
+			font-weight: 600;
+			font-size: 18px;
+			line-height: 22px;
+
+			&:hover {
+				text-decoration: underline;
+			}
+		}
+	}
+
+	&__summary {
+		font-size: 13px;
+		color: var(--gray);
+		visibility: hidden;
+		overflow: hidden;
+		max-height: 0;
+		position: relative;
+		top: -12px;
+	}
+
+	&__lessons.collapsed ~ &__summary {
+		visibility: visible;
+		max-height: 50px;
+		transition-delay: 150ms;
+	}
+
+}
+
+.sensei-lms-course-navigation-lesson {
+
+	display: flex;
+	align-items: center;
+
+	&__link {
+		flex: 1;
+		display: flex;
+		align-items: center;
```

Comment:
In TwentyTwenty One it's adding an underline. I think we could add a `text-decoration: none` here to avoid it. ;)

Reply:
Also fixed by 6b6b77e